changelog.md

```diff
@@ -1,3 +1,62 @@
+# 4.2.0 / 2022-04-29
+
+### New Features
+
+For users:
+- The user interface is now available in Arabic, using the new translation tool: Weblate
+- Handle right-to-left languages on interface
+- Feature improvement: Notifications, Kanban, Agenda
+- UI/UX improvement (responsive, buttons, redirections)
+- External links to pictures now have a preview in comments
+
+For admins:
```

Comment:
I'm not sure about this wording. Who is admin in Tracim ? (I can explain difference with you)